src/styles/elements/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("../../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars div:nth-child(-n + 4) {
+  background-image: url("../../images/star-active.svg");
+}
```

Comment:
reuse whole stars block from task with stars

Reply:
not fixed, it should work for any quantity of stars, don't use `div` selector